### ISSUE TITLE
ResourcesNotFoundException: Resource ID #0x7f070016 type #0x3 is not valid

### ISSUE THREAD

**Issue by ForceGT** [googlemaps/android-maps-compose#222]
Related stacktrace:

```
android.content.res.Resources$NotFoundException: Resource ID #0x7f070016 type #0x3 is not valid
                 	at android.content.res.Resources.getDimensionPixelSize(Resources.java:827)
                 	at com.google.maps.api.android.lib6.impl.ax.d(:com.google.android.gms.dynamite_mapsdynamite@214218120@21.42.18 (190800-0):5)
                 	at com.google.maps.api.android.lib6.impl.fh.<init>(:com.google.android.gms.dynamite_mapsdynamite@214218120@21.42.18 (190800-0):6)
                 	at com.google.maps.api.android.lib6.impl.bo.az(:com.google.android.gms.dynamite_mapsdynamite@214218120@21.42.18 (190800-0):16)
                 	at com.google.maps.api.android.lib6.impl.cz.d(:com.google.android.gms.dynamite_mapsdynamite@214218120@21.42.18 (190800-0):2)
                 	at com.google.android.gms.maps.internal.q.aX(:com.google.android.gms.dynamite_mapsdynamite@214218120@21.42.18 (190800-0):14)
                 	at dx.onTransact(:com.google.android.gms.dynamite_mapsdynamite@214218120@21.42.18 (190800-0):4)
                 	at android.os.Binder.transact(Binder.java:1064)
                 	at com.google.android.gms.internal.maps.zza.zzc(com.google.android.gms:play-services-maps@@18.0.0:2)
                 	at com.google.android.gms.maps.internal.zzl.onCreate(com.google.android.gms:play-services-maps@@18.0.0:3)
                 	at com.google.android.gms.maps.zzag.onCreate(com.google.android.gms:play-services-maps@@18.0.0:3)
                 	at com.google.android.gms.dynamic.zac.zab(com.google.android.gms:play-services-base@@18.0.1:1)
                 	at com.google.android.gms.dynamic.zaa.onDelegateCreated(com.google.android.gms:play-services-base@@18.0.1:3)
                 	at com.google.android.gms.maps.zzah.zzb(com.google.android.gms:play-services-maps@@18.0.0:5)
                 	at com.google.android.gms.maps.zzah.createDelegate(com.google.android.gms:play-services-maps@@18.0.0:1)
                 	at com.google.android.gms.dynamic.DeferredLifecycleHelper.zaf(com.google.android.gms:play-services-base@@18.0.1:6)
                 	at com.google.android.gms.dynamic.DeferredLifecycleHelper.onCreate(com.google.android.gms:play-services-base@@18.0.1:1)
                 	at com.google.android.gms.maps.MapView.onCreate(com.google.android.gms:play-services-maps@@18.0.0:4)
                 	at com.google.maps.android.compose.GoogleMapKt.lifecycleObserver$lambda-11(GoogleMap.kt:191)
                 	at com.google.maps.android.compose.GoogleMapKt.$r8$lambda$ml76XdpXTOPqW4q28tKk_7GmLZk(Unknown Source:0)
                 	at com.google.maps.android.compose.GoogleMapKt$$ExternalSyntheticLambda0.onStateChanged(Unknown Source:4)
                 	at androidx.lifecycle.LifecycleRegistry$ObserverWithState.dispatchEvent(LifecycleRegistry.java:360)
                 	at androidx.lifecycle.LifecycleRegistry.addObserver(LifecycleRegistry.java:202)
                 	at com.google.maps.android.compose.GoogleMapKt$MapLifecycle$1.invoke(GoogleMap.kt:172)
                 	at com.google.maps.android.compose.GoogleMapKt$MapLifecycle$1.invoke(GoogleMap.kt:168)
                 	at androidx.compose.runtime.DisposableEffectImpl.onRemembered(Effects.kt:81)
                 	at androidx.compose.runtime.CompositionImpl$RememberEventDispatcher.dispatchRememberObservers(Composition.kt:1091)
                 	at androidx.compose.runtime.CompositionImpl.applyChangesInLocked(Composition.kt:818)
                 	at androidx.compose.runtime.CompositionImpl.applyChanges(Composition.kt:839)
                 	at androidx.compose.runtime.Recomposer.composeInitial$runtime_release(Recomposer.kt:978)
                 	at androidx.compose.runtime.ComposerImpl$CompositionContextImpl.composeInitial$runtime_release(Composer.kt:3848)
                 	at androidx.compose.runtime.ComposerImpl$CompositionContextImpl.composeInitial$runtime_release(Composer.kt:3848)
                 	at androidx.compose.runtime.ComposerImpl$CompositionContextImpl.composeInitial$runtime_release(Composer.kt:3848)
                 	at androidx.compose.runtime.CompositionImpl.setContent(Composition.kt:519)
                 	at androidx.compose.ui.layout.LayoutNodeSubcompositionsState.subcomposeInto(SubcomposeLayout.kt:468)
                 	at androidx.compose.ui.layout.LayoutNodeSubcompositionsState.subcompose(SubcomposeLayout.kt:441)
                 	at androidx.compose.ui.layout.LayoutNodeSubcompositionsState.subcompose(SubcomposeLayout.kt:432)
                 	at androidx.compose.ui.layout.LayoutNodeSubcompositionsState.subcompose(SubcomposeLayout.kt:421)
```

**Comment by DSteve595** [comment]
Can you provide more info, such as a sample project, repro steps, device and OS info?

**Comment by ForceGT** [comment]
@DSteve595 This bug randomly happens
I see an issue opened in official google-maps-utils as well


https://github.com/googlemaps/android-maps-utils/issues/532

**Comment by DSteve595** [comment]
@ForceGT It's hard to investigate this issue without more info. The most helpful thing would be for you to share a small project that reproduces this bug. If that's not possible, please share more info about the dependencies you're using (including their versions), and what your code is doing.

**Comment by realh** [comment]
I've been experiencing this error too. I found a [stackoverflow post](https://stackoverflow.com/questions/72538632/android-content-res-resourcesnotfoundexception-resource-id-0x7f070016-type-0) about it. The suggestion to clear the cache or uninstall the app works for me, but the error can come back. Unfortunately I can't provide a simple example. I looked up 0x7f070016 in resources.arsc in the apk and it says it's drawable/abc_ic_arrow_drop_right_black_24dp. I don't reference that anywhere in my own code. It looks like the common factor is using Google Maps. The bug may be in the underlying library though, not Compose.
